packages/editor/src/managers/clipboard/paste-manager.ts

```diff
@@ -198,7 +191,10 @@ export class PasteManager {
         selectedBlock &&
           this._addBlocks(blocks[0].children, selectedBlock, 0, addBlockIds);
         //This is a temporary processing of the divider block, subsequent refactoring of the divider will remove it
-        if (blocks[0].flavour === 'affine:divider') {
+        if (
+          blocks[0].flavour === 'affine:divider' ||
```

Comment:
``` ts
matchFlavours(blocks[0], ['affine:divider', 'affine:embed'])
```

Reply:
I tried to do it, but I found block[0]'s type is not BaseMoudel, that is OpenBlockInfo, so I can't use this function